Tools/Pipeline/MainWindow.cs

```diff
@@ -66,6 +66,10 @@ public void Attach(IController controller)
 
             cmdDebugMode.Checked = PipelineSettings.Default.DebugMode;
             CmdDebugMode_Executed(this, EventArgs.Empty);
+            
+            // MultithreadMode is experimental and disabled on startup.
+            //cmdMultithreadMode.Checked = PipelineSettings.Default.MultithreadMode; 
```

Comment:
Well, I though it would be better since we just test it out but after using it for a few times I too don't see a point since we clearly state it's experimental.